sidebars.js

```diff
@@ -839,6 +839,7 @@ module.exports = {
             "self-managed/platform-deployment/troubleshooting",
           ],
         },
+        "self-managed/platform-deployment/amazon-aurora-postgresql",
```

Comment:
This is at the wrong level in the hierarchy. However, we don't have a great space for it with existing docs. 

Do we have other database config in the works? Would it make sense to have a dedicated section for configuring different databases cross components?

Reply:
I do think this would make sense, especially given the cross-component duplication we've been seeing.

I think this requires a separate issue though, to go through the existing documentation and outline what content could live in such a section (would it be too lean? should we leave as-is until we see more? do users like this new layout, or does it leave them more confused in terms of navigation across the docs?)

Reply:
I would be ok with this as a separate issue, but I'm still not happy with the hierarchy as-is. @Ben-Sheppard is the Identity team authoring any additional database config? 

Would it make sense to move this into Helm/Kubernetes > Guides or Operational guides? I want the 3 deployment options very clear in the installation section.

Reply:
We aren't at the moment (I'm not sure if Web Modeler are though?)- The only thoughts I have regarding the location is something that I've felt with other docs I've written/reviewed which is, the database config is really independent of the installation method IMO so I could use AWS Aurora in all Helm/K8s, Docker, and Manual (referenced from our current docs structure).

Reply:
> (I'm not sure if Web Modeler are though?)

Neither are we.

> the database config is really independent of the installation method IMO so I could use AWS Aurora in all Helm/K8s, Docker, and Manual (referenced from our current docs structure).

Agree - I'd not put the AWS configuration below "Helm/Kubernetes" because of this reason, there should be a more generic location for cross-component configuration

Reply:
@christinaausley let's discuss in our 1:1 this week to see if we can make a quick change here to the content structure.